.circleci/config.yml

```diff
@@ -222,20 +225,15 @@ workflows:
               pytest-group: [ classification, misc ]
           requires:
             - ci-base-<< matrix.python-version >>
-      - example-test:
+      - example-test-large:
```

Comment:
Nit: I think this can be simplified into

```
workflows:
  ci:
    jobs:
      ...
      -- example-test:
             name: example-test-large
             <unchanged>
       -- example-test:
              name: example-test-small
              <unchanged>
```

with
```
jobs
  example-test:
     <unchanged-from-example-test-large>
```